<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Semi_Mode</class>
 <widget class="QDialog" name="Dialog_Semi_Mode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>203</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>279</width>
    <height>203</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>279</width>
    <height>203</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Semi-Auto Mode Confirm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Distinguished Cube Color: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_color_result">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Do you want to continue?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yes">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Other operations:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_red">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Place Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_green">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Place Green</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_blue">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Place Blue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
